buildbot.tac

```diff
@@ -23,7 +26,3 @@ from flocker_bb.upgrade import UpgradeService
 
 master = BuildMaster(basedir, configfile)
 UpgradeService(basedir, configfile, master).setServiceParent(application)
-
-
-from twisted.application.internet import TimerService
-TimerService(30, master.botmaster.maybeStartBuildsForAllBuilders).setServiceParent(master)
```

Comment:
Was this a less successful work-around for the issue of slaves/builds sometimes not starting/running?  Which is now presumably no longer necessary?